### PR TITLE
Make header compatible with georchestra Security Proxy

### PR DESCRIPTION
**Pull request by jahow**
Login and logout URLs used to be only compatible with the new gateway.

### CODE REVIEW

File: vite.config.js [hunk 1]
```diff
@@ -19,10 +19,8 @@ export default defineConfig({
   },
   server: {
     proxy: {
-      '^/(whoami|login|logout)': {
-        // FIXME: replace this with demo.georchestra.org when it uses the gateway
-        target:
-          'https://gateway.mel.integration.apps.gs-fr-prod.camptocamp.com',
+      '^/(whoami|\\?login|logout)': {
+        target: 'https://mel.integration.apps.gs-fr-prod.camptocamp.com',
```
[the remark this answers is not in the document]
Comment (jahow) [reply vite.config.js:23]: this is just for dev purposes; ideally we should point at demo.georchestra.org but I think some endpoints are not working as expected there.

[the remark this answers is not in the document]
Comment (landryb) [reply vite.config.js:23]: > ideally we should point at demo.georchestra.org but I think some endpoints are not working as expected there.

which ones ? demo.geor.org is not updated automagically, so if you tell me which part should be working and how, i can have a look

[the remark this answers is not in the document]
Comment (jahow) [reply vite.config.js:23]: I found these ones:
* https://demo.georchestra.org/?login will not take me to the login page
* https://demo.georchestra.org/whoami does not respond

**Review by tkohr — APPROVED** [review]
Thanks for the fix @jahow !

**Comment by landryb** [comment]
> @landryb I'm going to merge this as is, as soon as we managed to fix the issues with demo.georchestra.org we can update the dev settings. Thanks

given https://github.com/georchestra/header#requirements i think we can just give up on that for now ? :) i've updated demo pkgs with latest for master for the S-P, but i wouldnt know where to update the header from.

**Pull request merged**